cves/kernel/CVE-2020-26088.yml

```diff
@@ -391,37 +407,39 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      allowing least-privilege processes to create these sockets was a mistake. adding levels of privilege to do certain functions was added.
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
     note:
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
+    applies: false
     note:
   secure_by_default:
-    applies:
+    applies: false
     note:
   yagni:
-    applies:
-    note:
+    applies: true
+    note: |
+      My belief is that the priv check was only added because the devs did not expect processes to leverage the lack of check.
```

Comment:
Don't shorten the word privilege here

Reply:
Or developers.

Reply:
I don't even know why I did that. Thanks for pointing it out, its fixed.